tst/standard/cong.tst

```diff
@@ -376,21 +376,21 @@ gap> EquivalenceRelationCanonicalLookup(ccong);
 gap> cong = ccong;
 true
 
-# Two infinite congruences (will try next method)
-gap> F := FreeSemigroup(2);;
-gap> cong1 := ReesCongruenceOfSemigroupIdeal(SemigroupIdeal(F, [F.1]));;
-gap> cong2 := SemigroupCongruence(F, [F.1, F.1^2]);;
-gap> cong1 = cong2;
-false
-gap> F := FreeSemigroup(2);;
-gap> cong1 := LeftSemigroupCongruence(F, [F.1, F.2]);;
-gap> cong2 := RightSemigroupCongruence(F, [F.1, F.2]);;
-gap> cong1 = cong2;
-Error, no method found! For debugging hints type ?Recovery from NoMethodFound
-Error, no 2nd choice method found for `ImagesSet' on 2 arguments
-gap> cong2 = cong1;
-Error, no method found! For debugging hints type ?Recovery from NoMethodFound
-Error, no 2nd choice method found for `ImagesSet' on 2 arguments
+# These examples would run forever trying to calculate EquivalenceRelationPartition
+#gap> F := FreeSemigroup(2);;
+#gap> cong1 := ReesCongruenceOfSemigroupIdeal(SemigroupIdeal(F, [F.1]));;
+#gap> cong2 := SemigroupCongruence(F, [F.1, F.1^2]);;
+#gap> cong1 = cong2;
+#false
```

Comment:
I removed this for valid reasons, which I've now actually explained in the test file.  Basically, this test was invalid and was passing because of luck.  My new version makes this much clearer.